fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerMutualTest.java

```diff
@@ -451,6 +455,132 @@ void testMutualIndexingCrashAndAllowContinueNonMutually(boolean explicit) {
         assertAllValidated(indexes);
     }
 
+    @ParameterizedTest
+    @BooleanSource
+    void testMultiTargetContinueAsMutual(boolean explicit) {
```

Comment:
It seems like there should also be tests of multi to-and-from mutual where they have different, potentially overlapping index sets.

Reply:
✅